api/tests/routes/test_detection.py

```diff
@@ -5,7 +5,7 @@
 from doctr.utils.metrics import box_iou
 
 
-@pytest.mark.asyncio
+@pytest.mark.asyncio()
```

Comment:
Why the parenthesis ?

Reply:
formated from ruff

Reply:
I have some doubt about this one. Even in `pytest-asyncio` it's written without parenthesis: https://pytest-asyncio.readthedocs.io/en/latest/
It can have also another meaning :thinking:
Would it be possible to ignore this one ?

Reply:
Will update this tomorrow and give you a ping :)